lighthouse-core/report/templates/report-template.html

```diff
@@ -177,3 +176,13 @@ <h1 class="menu__header-title">Lighthouse</h1>
     </footer>
   </section>
 </div>
+
+{{#if scripts }}
+<script>
+  window.lhresults = {{{lhresults}}};
```

Comment:
Really not trying to be argumentative, this just seems like an easy way to secure us for the future :)